tests/test-extensions/local_extension/cookiecutter.json

```
@@ -2,8 +2,11 @@
   "project_slug": "Foobar",
   "test_value_class_based": "{{cookiecutter.project_slug | foobar}}",
   "test_value_function_based": "{{cookiecutter.project_slug | simplefilterextension}}",
+  "test_value_function_test_pos_based": "{% if 'XAX' is palindrome%}PASS1{%else%}PASS1{% endif %}",
+  "test_value_function_test_neg_based": "{% if 'XAY' is palindrome%}FAIL2{%else%}PASS2{% endif %}",
```

These tests are not correct and are not clear (`PASS1` is duplicated, for example).

Please change these lines to something like this, update the test suite to look for `"XX=palindrome"` and `"XY=text"`, and then confirm that the test suite passes:

```suggestion
  "test_value_function_test_pos_based": "XX={% if 'XX' is palindrome%}palindrome{%else%}text{% endif %}",
  "test_value_function_test_neg_based": "XY={% if 'XY' is palindrome%}palindrome{%else%}text{% endif %}",
```

---

I would really like to have an excuse for this, but looking for it, all I find is embarrassment 😊